Gemfile

```diff
@@ -24,6 +24,7 @@ group :development, :test do
 end
 
 group :development do
+  gem 'blueprinter'
```

Comment:
Blueprinter is the gem you are going to use to render the response to whoever "speaks" to your api. So you aren’t going to use it just in development. Add it as you would add rails or devise.